gulp/tasks/test_debug.js

```diff
@@ -0,0 +1,26 @@
+(function () {
+    'use strict';
+
+    var gulp = require('gulp'),
+        karma = require('karma').server,
+        path = require('path'),
+        paths = {
+            spec: [
+                'test/specs/**/*.js'
+            ],
+            karamaConf: 'karma.debug.conf.js'
+        };
+
+    // starts up Karma using Chrome in debug mode
+    function runKarma(configFile, cb) {
```

Comment:
Might just be ignorance on my part, but what kind of object is `cb`? Should that be clearer?